relay/sources/rudder-pkg/Readme.md

```diff
@@ -0,0 +1,122 @@
+| 8.0.1-2.9-nightly         | 8.0.1                    | ❌             |
+| 8.0.1~rc3-2.9             | 8.0.1                    | ❌             |
+| 8.0.1~rc3-2.9             | 8.0.1~rc3                | ✅             |
+| 8.0.2~rc3-2.9             | 8.0.2~rc2                | ❌             |
+| 8.0.2~beta1-2.9           | 8.0.2~beta1              | ✅             |
+| 8.0.2~alpha1.2-2.9        | 8.0.2\~alpha2~git12345    | ❌             |
+
+
+## Package scripts
+
+Package scripts are executables that are run at specific timing during the installation/uninstallation process.
+They are expected to return 0, otherwise the installation/uninstallation will be aborted.
+
+They **must** be packaged inside the `scripts.txz` archive in the rpkg. And they will always be unpacked on the Rudder server under `/var/rudder/packages/rudder-plugin-<plugin name from metadata>`.
+
+## Plugin content
+
+A plugin can contain multiple archives that will be unpacked on different locations on the system at install time.
+Each file deployed this way will be removed when the plugin will be uninstalled.
+
+Each content archive must be listed in the metadata under the `contents` field, using the archive name as key, and the target installation path as value.
+
+```json
+  "content": {
+    "files.txz": "/opt/rudder/share",
+    "var_rudder.txz": "/var/rudder"
+  }
+```
+
+
+## RPKG installation flowchart
+
+A failure in any of the steps in the schema below will abort the plugin installation without cleaning.
+
+[![](https://mermaid.ink/img/pako:eNp1U0tz2jAQ_isanSHExpTEh840QBKSaQ-0PrR2D8JaQMWWNJJMQoD_XlmSmdCHDiut9vl9Wh1wKSjgFK8q8VJuiDLo27TgyC7dLNeKyA1ayO0aMa4NqSpimODe_ilfAKHIbADVYAglhvz0lrt8soFyi15gSaREpailjVuyipl9cJkEFwoSOAVeMtDBND3MNWIGEY4aaTugcPKGWT57NYqUBqm2I0nKLVkD0qVi0nTR9_mi4UgqaBsOtmB6uIwvBTfAO-NjnkkLARyeABYoklWzZhy14JZEQ3Ce-yJCm7-rPOUzTpYVoF9EoRWrwIJZWTAd8mdLm81uifb0hGvLwh-0Zzy0gURFQaEdKH0mP4vyKdOuTtvwf2plsS1Wi513UmCfz2LyjhXTrWLBtbZ_48yG7-MpUFa6DMF75Ruzp8GOqIFqqFUH4V26B8mS9zm6SCVqn_OyoGMhjBfq9z-iOycnToa5nDrl-EUcZxcX30EfsyiMiru6d_LByUcn504-OflcnLl0ehb7bei3xG22BO7hGlRNGLXf5NCGFNi2XkOBU3ukRG0LXPCT9SONEV_3vMSpUQ30sBLNeoPTFam01Ro3YFNG7PvW51tJ-A8h6i7EcmyE-uw_pfubzgWnB_yK02QYXyXJTZzE0c141MN7nPavr67bNYqj29toPIqi0YdxMh6eevjN5Y1OvwGl5T4S?type=png)](https://mermaid.live/edit#pako:eNp1U0tz2jAQ_isanSHExpTEh840QBKSaQ-0PrR2D8JaQMWWNJJMQoD_XlmSmdCHDiut9vl9Wh1wKSjgFK8q8VJuiDLo27TgyC7dLNeKyA1ayO0aMa4NqSpimODe_ilfAKHIbADVYAglhvz0lrt8soFyi15gSaREpailjVuyipl9cJkEFwoSOAVeMtDBND3MNWIGEY4aaTugcPKGWT57NYqUBqm2I0nKLVkD0qVi0nTR9_mi4UgqaBsOtmB6uIwvBTfAO-NjnkkLARyeABYoklWzZhy14JZEQ3Ce-yJCm7-rPOUzTpYVoF9EoRWrwIJZWTAd8mdLm81uifb0hGvLwh-0Zzy0gURFQaEdKH0mP4vyKdOuTtvwf2plsS1Wi513UmCfz2LyjhXTrWLBtbZ_48yG7-MpUFa6DMF75Ruzp8GOqIFqqFUH4V26B8mS9zm6SCVqn_OyoGMhjBfq9z-iOycnToa5nDrl-EUcZxcX30EfsyiMiru6d_LByUcn504-OflcnLl0ehb7bei3xG22BO7hGlRNGLXf5NCGFNi2XkOBU3ukRG0LXPCT9SONEV_3vMSpUQ30sBLNeoPTFam01Ro3YFNG7PvW51tJ-A8h6i7EcmyE-uw_pfubzgWnB_yK02QYXyXJTZzE0c141MN7nPavr67bNYqj29toPIqi0YdxMh6eevjN5Y1OvwGl5T4S)
```

Comment:
GitHub can render mermaid directly https://github.blog/developer-skills/github/include-diagrams-markdown-files-mermaid/

Reply:
(and most graphical text editors too)